server/courselist.rb

```diff
@@ -392,7 +392,7 @@ def self.vetoRequest(user, request_url)
     ## Make sure people only ask about themselves (or are privileged)
     vetoResult = CourseList.vetoRequest request.env['REMOTE_USER'], request.env['REQUEST_URI']
     logger.debug "vetoResult: "+vetoResult.to_s
-    halt 401 if vetoResult == true
+    halt 403 if vetoResult == true
```

Comment:
The change is obvious. However, based on the definition of 401 vs 403, I still think 401 is more proper...

401 Unauthorized
    Similar to 403 Forbidden, but specifically for use when authentication is required and has failed or has not yet been provided. The response must include a WWW-Authenticate header field containing a challenge applicable to the requested resource. See Basic access authentication and Digest access authentication.
403 Forbidden
    The request was a valid request, but the server is refusing to respond to it. Unlike a 401 Unauthorized response, authenticating will make no difference.